### PR TITLE
ports TG's HF blade

### PR DESCRIPTION
**Pull request by 1393F**
## About The Pull Request
https://github.com/tgstation/tgstation/pull/63861

I can't figure out how to fix this

## Changelog
:cl:
tweak: the HF blade has been rebalanced
/:cl:

### CODE REVIEW

**Comment by thevandie** [comment]
everything is fucked

**Comment by 1393F** [comment]
> everything is fucked

those are just checks mate, only got 2 undefined things that I can't find replacements for

**Comment by thevandie** [comment]
those sure seem like important checks now, 2 months later.

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -1028,19 +1062,75 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+	if(isliving(target))
+		var/mob/living/living_target = target
+		living_target.apply_damage(force*damage_mod, BRUTE, sharpness = SHARP_EDGED, wound_bonus = wound_bonus, bare_wound_bonus = bare_wound_bonus, def_zone = user.zone_selected)
+		log_combat(user, living_target, "slashed", src)
+		if(living_target.stat == DEAD && prob(force*damage_mod*0.5))
+			living_target.visible_message(span_danger("[living_target] explodes in a shower of gore!"), blind_message = span_hear("You hear organic matter ripping and tearing!"))
+			living_target.gib()
+			log_combat(user, living_target, "gibbed", src)
+	else if(target.uses_integrity)
+		target.take_damage(force*damage_mod*3, BRUTE, MELEE, FALSE, null, 50)
```
Comment (EgorDinamit) [inline code/game/objects/items/weaponry.dm:1091]:
```suggestion
	else if(isobj(target))
		var/obj/obj_target = target
		obj_target.take_damage(force*damage_mod*3, BRUTE, MELEE, FALSE, null, 50)
```

Comment (EgorDinamit) [reply code/game/objects/items/weaponry.dm:1091]: Here's your fix.

**Pull request merged**